designer/server/src/common/components/pagination/template.test.js

```diff
@@ -0,0 +1,580 @@
+import { renderMacro } from '~/test/helpers/component-helpers.js'
+
+describe('Pagination Component', () => {
+  /** @type {Element | null} */
+  let $pagination = null
```

Comment:
If we use role queries ([see comment](https://github.com/DEFRA/forms-designer/pull/616/files#r1875775788)), we don't need to handle `null` 💪 

```suggestion
  /** @type {HTMLElement} */
  let $pagination
```

Reply:
Not sure if we can do this, as there might be times when pagination isn't loaded at all e.g. only one page

Reply:
That's fair. We can still use the role queries though? Just with the `| null` type back in

Reply:
Managed to find a workaround for this, so we can keep it as you suggested 👍